bedrock/base/templates/macros.html

```diff
@@ -377,6 +377,16 @@
 </header>{#-- /#fxosnav-header --#}
 {%- endmacro %}
 
+{% macro google_play_button(application='firefox', class_name='', extra_attributes=None, id='', include_rel=True, target_blank=False) -%}
+{% set optional_attributes={'alt': _('Get it on Google Play'), 'width': '152', 'height': '45', 'l10n': True} %}
+{% if extra_attributes %}
+  {% do optional_attributes.update(extra_attributes) %}
+{% endif %}
+<a{% if class_name %} class="{{ class_name }}"{% endif %}{% if id %} id="{{ id }}"{% endif %}{% if include_rel %} rel="external"{% endif %} href="{% if application == 'stumbler' %}https://play.google.com/store/apps/details?id=org.mozilla.mozstumbler{% else %}{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}{% endif %}"{% if target_blank %} target="_blank"{% endif %}{% if application == 'firefox' %} data-link-type="download" data-download-os="Android" data-mozillaonline-link="{{ settings.GOOGLE_PLAY_FIREFOX_LINK_MOZILLAONLINE }}"{% endif %}>
```

Comment:
There's a lot of logic here that's a bit difficult to read. Could we maybe pass the `href` as a parameter to the macro, that defaults to Firefox for Android instead, avoiding the long `application` conditional? Thoughts?